packages/ragbits-core/tests/unit/vector_stores/test_chroma.py

```diff
@@ -101,6 +101,15 @@ async def test_retrieve(
         assert entry.key == result["content"]
 
 
+async def test_remove(mock_chromadb_store: ChromaVectorStore) -> None:
+    ids_to_remove = ["1c7d6b27-4ef1-537c-ad7c-676edb8bc8a8"]
+
+    await mock_chromadb_store.remove(ids_to_remove)
+
+    mock_chromadb_store._client.get_or_create_collection().delete.assert_called_once()  # type: ignore
+    mock_chromadb_store._client.get_or_create_collection().delete.assert_called_with(ids=ids_to_remove)  # type: ignore
```

Comment:
IMO you should just list the ids available with `list` method instead of checking whether we call specific methods of a *private* field. This is just my opinion though, I'd love to hear about it from the rest of the team.

This remark applies to all unit tests.